app/routes.py

```diff
@@ -127,37 +89,25 @@ def get_kml_metadata(kml_id):
 def update_kml(kml_id):
     db = get_db()
 
-    item = db.get_item(kml_id)
-    admin_id = validate_permissions(item)
+    db_item = db.get_item(kml_id)
+    admin_id = validate_permissions(db_item)
+
+    # Get the client version
+    author_version = request.form.get('author_version', None)
```

Comment:
would it make sense to define the absence of author_version in the DB to the default value set in another file above?
There's no real benefit in knowing that this KML was generated before we add this new column, as both cases will be considered dangerous to modify by the new viewer as long as it is not productive.

Reply:
Ho, that's the update route no? So if no author version is defined, we do not change anything, is that it?